e2e/__tests__/toMatchInlineSnapshot.test.ts

```diff
@@ -360,3 +360,24 @@ test('multiple custom matchers and native matchers', () => {
   expect(exitCode).toBe(0);
   expect(wrap(fileAfter)).toMatchSnapshot('multiple matchers');
 });
+
+test('indentation is correct in the presences of existing snapshots', () => {
+  const filename = 'existing-snapshot.test.js';
+  const test = `
```

Comment:
```suggestion
  const test = dedent`
```

should make the snapshot in the test better? I don't know if we depend on it, but just add it if not

Reply:
No we don't depend on it - Happy to add it, but can also reproduce the bug using a single line string (i.e `expect("hello").toMatchInlineSnapshot(``"hello"``)`), so can do either 🤷‍♂

Reply:
I think that would make the test clearer, when it still has weird indentation  it doesn't _look_ fixed, even though it is

Reply:
Agreed. `dedent` worked, so I've pushed that :)

Reply:
push it?

Reply:
Yeah apologizes ssh has decided it doesn't like the looks the internet has been giving it, and so refusing to connect.

I'll get it pushed in a few minutes

Reply:
Luckly `cmd.exe`'s `ssh` worked 🤷‍♂

Reply:
huh, weird. Thanks, looks great! 🙂 I opened up #9525, it seems to clean up things quite a bit

Reply:
Ah just realised I didn't install the `@types`, but it all works as they're already installed.

Want me to add them in this PR, or happy to have them added via yours? (since it'll trigger a whole new build for this PR 😬)

Reply:
I merged it to master and into this branch now 👍